examples/example_evo_regression.py

```diff
@@ -1,122 +1,183 @@
 import numpy as np
 import matplotlib.pyplot as plt
+import os
 import scipy.constants
 import sympy
-from sympy.printing.dot import dotprint
+from functools import partial
 import sys
 import time
 import torch
 
+from sympy.printing.dot import dotprint
+
 sys.path.insert(0, '../')
```

Comment:
i think we can remove this if the package is pip installable

Reply:
I will remove all of these calls from the scripts, incl. the test suite, once we have merged #19 . Let's do this in a separate PR.

Reply:
Added a ticket for this: #26